### PR TITLE
Updated PCA Tutorial

### LINKED ISSUE #1378: PCA Tutorial Needs Update
In PCA tutorial,
``` python
import MDAnalysis as mda
import MDAnalysis.analysis.pca as pca
from MDAnalysis.tests.datafiles import PSF, DCD

u = mda.Universe(PSF, DCD)
PSF_pca = pca.PCA(u, select='backbone')
PSF_pca.run()

n_pcs = np.where(PSF_pca.cumulated_var > 0.95)[0][0]
```
should return cumulated variances, instead returns
```python
AttributeError: 'PCA' object has no attribute 'cumulated_var'
```
# Current Version of MDAnalysis
0.16.0.dev0

### PR DESCRIPTION
**Pull request by kaceyreidy**
Fixes #1378 

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [X] Docs?
 - [ ] CHANGELOG updated?
 - [X] Issue raised/referenced?

### CODE REVIEW

**Comment by kaceyreidy** [comment]
@orbeckst Please review.

File: package/MDAnalysis/analysis/pca.py [hunk 1]
```diff
@@ -87,7 +87,7 @@
 ``cumulated_variance``. The value at the ith index of `cumulated_variance`
 is the sum of the variances from 0 to i.
 
-    >>> n_pcs = np.where(PSF_pca.cumulated_var > 0.95)[0][0]
+    >>> n_pcs = np.where(PSF_pca.cumulated_varance > 0.95)[0][0]
```
Comment (orbeckst) [inline package/MDAnalysis/analysis/pca.py]: Is it really "varance" and not "variance"?? Please double check!

**Comment by orbeckst** [comment]
@kaceyreidy If you need to make changes, just keep pushing to your branch. The updates will automatically show up in this PR.

File: package/MDAnalysis/analysis/pca.py (hunk 1 above)
Comment (kaceyreidy) [inline package/MDAnalysis/analysis/pca.py]: Ah I realize exactly what I did too. I forgot to hit the "I" key twice in vim.

**Comment by orbeckst** [comment]
@kaceyreidy please also add yourself to the file [`package/AUTHORS`](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS)

**Comment by richardjgowers** [comment]
@kaceyreidy congrats on your first PR!  Next time a line in CHANGELOG is usually required, but I'll sort it this time.

**Comment by orbeckst** [comment]
Sorry, I forgot to ask for the changelog, thanks for fixing it @richardjgowers

**Comment by kaceyreidy** [comment]
Oh, I'm sorry. Thank you for fixing it.

**Pull request merged**